### ISSUE TITLE
Azure core extension: Show a recent and favorites list on top

### ISSUE THREAD

**Issue by qifahs** [microsoft/azuredatastudio#7913]
Issue Type: <b>Feature Request</b>

Everytime I use the Azure core tree, I am usually navigating to the same server/db. It would be really helpful if I can see my recent items on top in a "recent" accordion. Or even be able to favorite/pin items in the tree so that they show up in a separate list on top.

Azure Data Studio version: azuredatastudio 1.13.0 (05a53af085861999ed8dc99596cd55829387d490, 2019-10-22T07:41:49.347Z)
OS version: Windows_NT x64 10.0.18362

**Comment by yorkshirelandscape** [comment]
Yes! This, please. I'm less high on the Recent list (though it would be a good option to have), but the ability to flag a sql object as a favorite and filter or sort by that property would be tremendous. That could be either within its natural grouping or duplicated at the top level of the server. Or even above the server level, if you're feeling daring. I can't believe the idea had never occurred to me until today, so I had to find out whether it had been suggested before and, lo! It has. Please make this happen.
